mypy/checkexpr.py

```diff
@@ -204,7 +204,7 @@ def visit_call_expr(self, e: CallExpr, allow_none_return: bool = False) -> Type:
                         or isinstance(typ, NameExpr) and node and node.kind == nodes.TYPE_ALIAS):
                     self.msg.type_arguments_not_allowed(e)
         self.try_infer_partial_type(e)
-        callee_type = self.accept(e.callee)
+        callee_type = self.accept(e.callee, disallow_any=False)
```

Comment:
This allows us to call function that are one of:
* unresolved
* unannotated or take `Any` as a parameter

Should we disallow this? My general thinking is that we want to make `--disallow-any=expr` easy to enable per-module. That way you can turn the option on for current module even if other modules are not well annotated.

As for functions that take explicitly annotated `Any` parameters, I think in a world with perfect stubs, we would disallow calling those. However, that's not the case (at least now) and, for instance, `print` function accept arguments of type `Any`. This would mean that you can't call `print` with `--disallow-any=expr` enabled.